addons/project/models/project_task.py

```diff
@@ -208,6 +207,8 @@ def _read_group_personal_stage_type_ids(self, stages, domain):
     partner_id = fields.Many2one('res.partner',
         string='Customer', recursive=True, tracking=True, compute='_compute_partner_id', store=True, readonly=False,
         domain="['|', ('company_id', '=?', company_id), ('company_id', '=', False)]", )
+    # Need this field to check there is no email loops when Odoo reply automatically
+    email_from = fields.Char('Email From')
```

Comment:
This field doesn't seem to be useful unless you have ``website_form`` installed. 

Maybe it would make sense to move it there ?